### PR TITLE
Payment resources

### PR DESCRIPTION
**Pull request by pedromtec**
#### What problem is this solving?

I need to add the queries and mutations of the checkout payment 

#### How should this be manually tested?

[Workspace](url)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: react/queries/cardSessionId.graphql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+query CardSessionId{
```
Comment (darlenedms) [inline react/queries/cardSessionId.graphql]:
```suggestion
query CardSessionId {
```

**Review by darlenedms — CHANGES_REQUESTED** [review]

**Review by darlenedms — APPROVED** [review]

**Review by omninando — DISMISSED** [review]
Where is the CHANGELOG?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
```
Comment (marcoskwkm) [inline CHANGELOG.md:10]: Why are we adding empty lines after each section header again?

File: react/mutations/savePaymentToken.graphql [hunk 3]
```diff
@@ -0,0 +1,5 @@
+mutation SavePaymentToken($paymentTokens: [PaymentToken]) {
+  savePaymentToken(paymentTokens: $paymentTokens) {
+    status
+  }
+}
```
Comment (marcoskwkm) [inline react/mutations/savePaymentToken.graphql]: nit: End each line with a `\n`.

File: react/package.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
     "ramda": "^0.26.1",
     "react": "^16.8.3",
     "react-dom": "^16.8.3",
-    "react-intl": "^2.7.2"
+    "react-intl": "3.9.1"
```
Comment (marcoskwkm) [inline react/package.json]: Why is this version fixed (without `^`)? Are you even using this dependency?

**Review by marcoskwkm — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (pedromtec) [reply CHANGELOG.md:10]: Prettier

**Review by marcoskwkm — APPROVED** [review]

**Pull request merged**